Description: Disable hard requirement on googleapi deps and oatuh2client
Author: Corey Bryant <corey.bryant@canonical.com>
Forwarded: not-needed

diff --git a/requirements.txt b/requirements.txt
index 0b1b472..78392f1 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -14,7 +14,6 @@ ipaddress>=1.0.7;python_version<'3.3' # PSF
 keystoneauth1>=2.14.0 # Apache-2.0
 keystonemiddleware!=4.5.0,>=4.2.0 # Apache-2.0
 lxml>=2.3 # BSD
-oauth2client!=4.0.0,>=1.5.0 # Apache-2.0
 oslo.config!=3.18.0,>=3.14.0 # Apache-2.0
 oslo.concurrency>=3.8.0 # Apache-2.0
 oslo.context>=2.9.0 # Apache-2.0
@@ -60,5 +59,4 @@ oslo.vmware>=2.11.0 # Apache-2.0
 os-brick>=1.8.0 # Apache-2.0
 os-win>=1.1.0 # Apache-2.0
 tooz>=1.28.0 # Apache-2.0
-google-api-python-client>=1.4.2 # Apache-2.0
 castellan>=0.4.0 # Apache-2.0
-- 
2.10.2

